feat: Add a completion component (#136)
tuist/Noora: docs/content/components/alerts/error.md

---
title: Error
titleTemplate: ":title · Alerts · Noora · Tuist"
description: A component that shows an error alert.
---

# Error Alert

> This component is a simple component that shows an error alert to the user.

Reviewer: ```suggestion
This component shows an error alert to the user.
```

| Property | Value |
| --- | --- |
| Interactivity | Non-required |

## Screenshot

> ![A screenshot that shows the looking of the error alert](/components/alert/error.png)

Reviewer: ```suggestion
![A screenshot with the error alert](/components/alert/error.png)
```

## API

### Example

```swift

> Noora().error("The project generation failed with.", next: [

Reviewer: ```suggestion
Noora().error("The project generation failed.", next: [
```

>     "Make sure the project manifest files are valid and compile",
>     "Ensure you are running the latest Tuist version",

Reviewer: ```suggestion
    "Make sure the project manifest files are valid and compile.",
    "Ensure you are running the latest Tuist version.",
```

I would try to stay consistent in our samples and add a dot at the end.
